test/spark/run-test.sh

```diff
@@ -3,18 +3,25 @@
 # run-test.sh - submit spark app after setup lakefs and upload dataset.
 #
 # uses the following environment variables:
+#   REPO - name of repository to create and use in test
 #   STORAGE_NAMESPACE - storage namespace used for repository creation
+#   USE_DIRECT_ACCESS - if set (to *anything*), use the direct thick Spark client
 # 
 # NOTE that this script should be run from the root project in order for docker compose to volume mount the project
 
 STORAGE_NAMESPACE=${STORAGE_NAMESPACE:-local://}
 
-docker-compose exec -T lakefs /app/wait-for localhost:8000
+export input="lakefs://${REPO}/main/sonnets.txt"
+export output="lakefs://${REPO}/main/sonnets-wordcount.csv"
```

Comment:
why do we need these as exported env variables - can we just pass the values without defining env variable for each data we pass?